src/python/pants/notes/2.7.x.md

```diff
@@ -1,5 +1,49 @@
 # 2.7.x Stable Releases
 
+## 2.7.0.dev4 (Aug 14, 2021)
```

Comment:
The date has ticked over but I'm happy letting the truly concerned reader dig a bit to figure things out.